### PR TITLE
Fix issue which prevented training when code stripping was enabled

### PR DESCRIPTION
**Pull request by surfnerd**
Fixes MLA-62.

Add a link.xml file to prevent the Grpc.Core.dll from being code stripped.

### CODE REVIEW

File: UnitySDK/Assets/ML-Agents/Plugins/ProtoBuffer/link.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<linker>
+    <assembly fullname="Grpc.Core" preserve="all"/>
+</linker>
```
Comment (chriselion) [inline UnitySDK/Assets/ML-Agents/Plugins/ProtoBuffer/link.xml]: EOL? if CI doesn't complain, no big deal.

Comment (surfnerd) [reply UnitySDK/Assets/ML-Agents/Plugins/ProtoBuffer/link.xml]: fixed

File: UnitySDK/Assets/ML-Agents/Plugins/ProtoBuffer/link.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<linker>
+    <assembly fullname="Grpc.Core" preserve="all"/>
```
Comment (chriselion) [inline UnitySDK/Assets/ML-Agents/Plugins/ProtoBuffer/link.xml]: Can you add a comment here? Something like
"Prevents Managed code stripping from removing code necessary for training. You may wish to disable this preservation (that is, re-enable code stripping) to reduce code size if you no longer need training. See also https://docs.unity3d.com/Manual/ManagedCodeStripping.html"

Just imagining someone trying to ship their game and things getting bloated by code they don't need.

Comment (surfnerd) [reply UnitySDK/Assets/ML-Agents/Plugins/ProtoBuffer/link.xml]: Done

**Pull request merged**